ansible/callback_plugins/datadog_callback.py

```diff
@@ -0,0 +1,280 @@
+# Source: https://github.com/DataDog/ansible-datadog-callback
```

Comment:
I don't like to include this in the repo, but I don't see a better method to source it

Reply:
`curl -LO ...`  of the raw versioned link on github? Though I am now sure that would be an improvement.

Could you please amend the link to be a versioned link - so that the exact version of the script is clear.

Reply:
Well, while downloading the source code will not make a difference in the travis workflow, it will add additional setup step for devops that use the repo manually (and during testing).

I prefer to keep the workflow as simple as `git clone ThisRepo; ansible-playbook ...`

Other solution would be to add https://github.com/DataDog/ as submodule and symlink the callback script in the ansible dir, still I don't see much improvement.

Reply:
Source URL changed in feb25216fafeace0dc5e1647ed4a8f28a5aedbf3